docs/ref/template-response.txt

```diff
@@ -102,9 +102,17 @@ Methods
         be extracted from ``content_type``, and if that is unsuccessful, the
         :setting:`DEFAULT_CHARSET` setting will be used.
 
-    .. versionadded:: 1.8
```

Comment:
I think I'd keep this at the current indent level and just have it at the bottom and mention both parameters, e.g. "The charset and using parameters were added."